.github/workflows/ci.yml

```diff
@@ -15,16 +15,20 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: Restore cached dependencies
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
           path: node_modules
           key: node-modules-${{ hashFiles('package.json') }}
       - name: Install dependencies
         run: npm install
       - name: Run Tests
         run: npm run ci
-      # - name: Upload coverage to Codecov
-      #   uses: codecov/codecov-action@v1
-      #   with:
-      #     file: ./coverage/coverage-final.json
-      #     token: ${{ secrets.CODECOV_TOKEN }}
+
+  automerge:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: fastify/github-action-merge-dependabot@v1
+        if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request' }}
+        with:
+          github-token: ${{secrets.github_token}}
```

Comment:
@simoneb I'm not sure if that secret exists as I don't have access to the project settings.

Reply:
the repo you got this from has an issue I guess. see https://docs.github.com/en/actions/reference/authentication-in-a-workflow

Reply:
Ahh, I see so `${{ secrets.GITHUB_TOKEN }}` is created automatically.

Reply:
more specifically, it is always available in a gh action run